app/code/Magento/InventoryApi/Test/_files/reservations_rollback.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+use Magento\Inventory\Model\ReservationCleanupInterface;
+use Magento\InventoryApi\Api\ReservationBuilderInterface;
+use Magento\InventoryApi\Api\ReservationsAppendInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+
+$reservationBuilder = Bootstrap::getObjectManager()->get(ReservationBuilderInterface::class);
+$reservationsAppend = Bootstrap::getObjectManager()->get(ReservationsAppendInterface::class);
+$reservationCleanup = Bootstrap::getObjectManager()->create(ReservationCleanupInterface::class);
```

Comment:
Replace on `get` method

Reply:
sorry, I don't understand what you mean

Reply:
We have `create` and `get` method in ObjectManager
In our case we do not need new instance each time
so we can use `Bootstrap::getObjectManager()->get(...)`  (as in code above)